### PR TITLE
Auto-upgrade CustomScript for Couchbase.

### PR DESCRIPTION
**Pull request by boumenot**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@boumenot__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (chrboum). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

File: couchbase-on-ubuntu/cluster-nodes-A2.json [hunk 1]
```diff
@@ -170,11 +170,12 @@
         "publisher": "Microsoft.OSTCExtensions",
         "type": "CustomScriptForLinux",
         "typeHandlerVersion": "1.2",
```
Comment (singhkay) [inline couchbase-on-ubuntu/cluster-nodes-A2.json]: Change this to 1.4 please

File: couchbase-on-ubuntu/cluster-nodes-A6.json [hunk 2]
```diff
@@ -210,11 +210,12 @@
         "publisher": "Microsoft.OSTCExtensions",
         "type": "CustomScriptForLinux",
         "typeHandlerVersion": "1.2",
```
Comment (singhkay) [inline couchbase-on-ubuntu/cluster-nodes-A6.json]: This to 1.4 as well

File: couchbase-on-ubuntu/cluster-nodes-D14.json [hunk 3]
```diff
@@ -450,11 +450,12 @@
         "publisher": "Microsoft.OSTCExtensions",
         "type": "CustomScriptForLinux",
         "typeHandlerVersion": "1.2",
```
Comment (singhkay) [inline couchbase-on-ubuntu/cluster-nodes-D14.json]: This to 1.4 as well

**Comment by singhkay** [comment]
@boumenot merge conflicts

**Comment by singhkay** [comment]
@boumenot Looks like more merge conflicts with the recent couchbase on ubuntu merge

**Comment by gatneil** [comment]
Sorry for the delay; more merge conflicts need to be fixed.

**Comment by gatneil** [comment]
No response; closing

**Pull request closed without merging**